derp/derp_server.go

```diff
@@ -162,6 +176,14 @@ type Server struct {
 	// src.
 	sentTo map[key.NodePublic]map[key.NodePublic]int64 // src => dst => dst's latest sclient.connNum
 
+	// flows is a map of all active flowStats on the server,
+	// protected by s.mu(). After allocation by the server,
```

Comment:
```suggestion
	// protected by s.mu. After allocation by the server,
```

(since `s.mu` isn't a function)